src/main/java/de/tum/in/www1/exerciseapp/domain/Result.java

```diff
@@ -41,6 +41,9 @@
     @Column(name = "score")
     private Long score;
 
+    @Column(name = "rated")
+    private Boolean rated;
```

Comment:
this change in the data model might be problematic for other types of exercises, please add a TODO in all other places assigned to me, that we should set this value to true or false and that we also need to check for null

Reply:
Actually, if we don't change anything for the other types of Exercises, they don't need this value at all and can just leave it at null. The only place where this could be problematic is in `ResultResource.getResultsForParticipation()`, but there I added the flag `ratedOnly` and set that flag when getting rated results for quiz exercises and ignore it when getting results for other types of exercises. Everywhere else, where this value is used, is already QuizExercise specific code (e.g. the statistics).

Reply:
Still we have to test that...